### PR TITLE
profile dropdown

### PR DESCRIPTION
**Pull request by sahdasamier**

### CODE REVIEW

File: src/components/ProfileDropdown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                        </Button>
+                    </Box>
+                </Menu>
+            </Box>
+
+        )
+    }
+}
+
+ProfileDropdown.PropTypes = {
```
Comment (ArnobChowdhury) [inline src/components/ProfileDropdown/index.tsx:123]: Proptypes are not necessary with typescript. Please remove these.

File: src/components/ProfileDropdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import {useNavigate} from "react-router-dom"
+import PropTypes from 'prop-types'
+const ProfileDropdown = ({isAuth} : any) => {
+    const navigate = useNavigate();
+    const handleLogout = () => {
+        localStorage.removeItem("accessToken");
+        localStorage.removeItem("refreshToken");
+        navigate("/login");
+    };
+    {
```
Comment (ArnobChowdhury) [inline src/components/ProfileDropdown/index.tsx:20]: Why this parenthesis?

**Review by ArnobChowdhury — COMMENTED** [review]
Please use prettier as your formatter. It is not properly formatted. Other than that, there are couple of changes I requested.

**Comment by sahdasamier** [comment]
screen shoot 
Screenshot (139)

**Pull request merged**